content/en/_index.Rmd

```diff
@@ -34,7 +33,10 @@ numofweeks <- "twoweek"
 site <- "CovidEmployeeData"
 ```
 
+`r end_transition_out`
```

Comment:
Is this the text edits you made in the csv? If yes they need to be pulled from the localization using the "insert_translation" function.

Reply:
Tried to address this in my most recent push